lib/tags.php

```diff
@@ -77,12 +77,3 @@ function splitVersion($version) {
 	$parts[3] = 0;
 	return $parts;
 }
-
-/**
- * @param string $versionString
- * @return bool
- */
-function isPreRelease($versionString)
-{
-	return preg_match('/[a-uw-z]/', $versionString); // @perf

```

Comment:
huh? thats a strange regex. why not `return strpos($versionString, ".pre") >= 0;`?